jvb/src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -272,26 +278,37 @@ private static boolean changeIsLargerThanThreshold(
      * belong to a particular {@link Endpoint}.
      */
     public BitrateController(
-            Endpoint destinationEndpoint,
+            String destinationEndpointId,
+            EventHandler eventHandler,
+            Supplier<List<T>> endpointsSupplier,
```

Comment:
There's a bit of inconsistency with the naming around 'endpoint' and 'media source container'

Reply:
Yeah, I left the variable naming as it is, because I imagine changing this in the future, and I don't know what the best option would be.